src/universal_agent/tgtg/cli.py

```
@@ -0,0 +1,744 @@
+# ── status ────────────────────────────────────────────────────────────────────
+
+def cmd_status(args):
+    creds = load_saved_credentials()
+    targets = load_targets()
+    high = [t for t in targets if t.desire == "high"]
+    watch = [t for t in targets if t.desire == "watch"]
+
+    print("── TGTG Sniper Status ──────────────────────────────")
+    print(f"Email:            {TGTG_EMAIL or '(not set)'}")
+    print(f"Saved tokens:     {'yes' if creds.get('access_token') else 'NO — run login first'}")
+    print(f"")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)